tests/test_single_topology_combined.py

```diff
@@ -94,29 +94,17 @@ def test_combined_parameters_nonbonded(host_system_fixture, hif2a_ligand_pair_si
 
         hgs = st.combine_with_host(host_sys, lamb, num_water_atoms)
         # check nonbonded terms
-        # 1) exclusions
-        # exclusions should be set for all ligand ixns in hgs.nonbonded_host_guest
-        hgs_exc_idxs = hgs.nonbonded_host_guest.potential.exclusion_idxs
-        hgs_exc_guest_idxs = set()
-        for i, j in hgs_exc_idxs:
-            if i > num_host_atoms or j > num_host_atoms:
-                assert (i, j) not in hgs_exc_guest_idxs
-                hgs_exc_guest_idxs.add((i, j))
-
-        expected_guest_idxs = set()
-
-        for i in range(st.get_num_atoms()):
-            for j in range(st.get_num_atoms()):
-                if i < j:
-                    expected_guest_idxs.add((i + num_host_atoms, j + num_host_atoms))
-
-        assert hgs_exc_guest_idxs == expected_guest_idxs
+        # 1) ligand ixns should be omitted in hgs.nonbonded_host_guest
+        hgs_atom_idxs = hgs.nonbonded_host_guest.potential.potentials[0].atom_idxs
```

Comment:
Might be good to make this test more robust by asserting the type of each of these potentials, since the potentials in the SummedPotential can change